src/py/mat3ra/periodic_table/element.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any, Dict
+
+from mat3ra.code.entity import InMemoryEntityPydantic
+from mat3ra.esse.models.element import ElementSchema, Symbol
+from pydantic import Field
+
+
+class ChemicalElement(ElementSchema, InMemoryEntityPydantic):
```

Comment:
We alreayd have element in made as https://github.com/Exabyte-io/made/blob/2dacf10eb4a12903743c091dc466a5c04c454d3d/src/js/basis/helpers.ts#L6. Where are we defining the concept at?